src/bin/Cicero.pl

```diff
@@ -145,7 +140,6 @@
 	-OPTIONS => $cap3_options
 );
 
-#print "blat_client_exe:$blat_client_exe\n\n";
```

Comment:
Depending on the value of `$blat_client_exe` at this point, this might actually be useful. We've had issues in the past where the `module load blat` default blat has changed and people were no longer using the correct version.

Reply:
It just contains the name of the executable to call, not the path. It is declared as `my $blat_client_exe = "gfClient";`